packages/bot-web-ui/src/components/dashboard/dashboard.tsx

```diff
@@ -29,12 +29,12 @@ const Dashboard = ({ active_tab, setActiveTab, toggleStrategyModal, is_drawer_op
     };
 
     return (
-        <>
+        <React.Fragment>
             <div className='dashboard__main'>
                 <div className='dashboard__container'>
-                    <ReactJoyride steps={DBOT_ONBOARDING} run={tour_run} continuous={true} showProgress={true} />
+                    <ReactJoyride steps={dbot_onboarding} run={tour_run} continuous={true} showProgress={true} />
```

Comment:
Can we pass booleans by this way?
```suggestion
                    <ReactJoyride steps={dbot_onboarding} run={tour_run} continuous showProgress />
```

Reply:
it has been removed one of our other PRs which is also in review. ✅